package.json

```diff
@@ -1,8 +1,17 @@
 {
   "private": true,
-  "workspaces": [
-    "packages/*"
-  ],
+  "workspaces": {
+    "packages": [
+      "packages/*"
+    ],
+    "nohoist": [
+      "@cloudflare/workers-types",
+      "@types/jest",
+      "@types/mocha",
+      "@types/node",
+      "@types/webpack-env"
+    ]
```

Comment:
Why this `nohoist` change? Unless it causing an issue, can we move to a separate PR?

Reply:
It was causing an issue I believe. I will verify it's really required now.

Reply:
it wasn't required. :/ ty